### PR TITLE
admin search: results are not highlighted #5982

### LINKED ISSUE #5982: admin search: results are not highlighted
The search keywords don't appear in the text box and neither are they highlighted in the results. I was searching for `"Chan Yi Jia"`
image

### PR DESCRIPTION
**Pull request by tarun14110**
Fixes #5982 
screenshot from 2016-08-14 06 20 17

### CODE REVIEW

**Comment by tarun14110** [comment]
ready for review

File: src/main/java/teammates/ui/controller/AdminSearchPageAction.java [hunk 1]
```diff
@@ -77,7 +78,7 @@ protected ActionResult execute() {
         }
               
         data.init();
-        
+        System.out.println("daataa" + data.searchKey);
```
Comment (chowyb) [inline src/main/java/teammates/ui/controller/AdminSearchPageAction.java]: Might want to remove this :)

**Comment by chowyb** [comment]
Feature seems to work as intended, but perhaps we could modify `InstructorSearchPageUiTest` to have a test for searcing for an exact string (via GodMode?)

**Comment by tarun14110** [comment]
@chowyb please have a look.

File: src/test/resources/pages/InstructorSearchPageSearchStudentsForStudent2WithExactString.html [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+                  </span>
+                  <span class="highlight">
+                    Course1
+                  </span>
+                </td>
+                <td class="align-center">
+                  Jo
+                  <span class="highlight">
+                    in
+                  </span>
```
Comment (chowyb) [inline src/test/resources/pages/InstructorSearchPageSearchStudentsForStudent2WithExactString.html]: Hmm I guess this feature still doesn't really work as intended then :stuck_out_tongue: ideally it shouldn't be highlighting stuff that isn't relevant to the search results, so highlighting `in` in `Joined` is not a good thing.

See if there's a way to highlight just the exact matches.

Comment (tarun14110) [inline src/test/resources/pages/InstructorSearchPageSearchStudentsForStudent2WithExactString.html]: Is it fine ? It highlights only the exact string while searching for exact string.
Can we have multiple exact string. I hadn't take care that (may be difficult to take care, as we cannot determine from where double quote started and where ended). Like `"student 2" "In Course2"`

**Comment by tarun14110** [comment]
@chowyb added a comment. Please have a look.

**Comment by chowyb** [comment]
@tarun14110 looks good now :+1:

**Comment by wkurniawan07** [comment]
@tarun14110 can you sync this branch with the latest `master`?

**Pull request merged**